### PR TITLE
[ENH] A little bit more explanation in README.rst.

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: letsencrypt/README.rst [hunk 1]
```diff
@@ -88,7 +89,21 @@ You'll also need a matching sudo configuration, like::
 
     your_odoo_user ALL = NOPASSWD: /usr/sbin/service nginx reload
 
-Further, if you force users to https, you'll need something like for nginx::
+The line above can be added to /etc/sudoers through the visudo command, or
+even better be put in a separate file in the /etc/sudoers.d directory. For
```
Comment (hbrunn) [inline letsencrypt/README.rst]: add '...if your distribution supports it' (`sudoers.d` is debian specific to my knowledge)

**Comment by hbrunn** [comment]
:+1:

**Comment by moylop260** [comment]
👍

File: letsencrypt/README.rst [hunk 2]
```diff
@@ -88,7 +89,21 @@ You'll also need a matching sudo configuration, like::
...
+instance in a file /etc/sudoers.d/letsencrypt. (Name should not end in ~ and
+should not contain a dot). This file should be owned by root and have
+authorities set by ``chown 0440 your_filename``.
```
Comment (hbrunn) [inline letsencrypt/README.rst]: as you give a linux admin course here anyways, propose to edit this file with `visudo -f`, this also creates it with correct permissions

File: letsencrypt/README.rst (hunk 1 above)
Comment (lasley) [inline letsencrypt/README.rst]: You are correct that this is a Debian default. Cent would need to implement via an include directive.

**Comment by lasley** [comment]
👍

**Comment by NL66278** [comment]
Rewrote the part on using the /etc/sudoers.d directory. By strongly suggesting to use letsencrypt filename no need to talk about naming conventions for this file.

**Comment by hbrunn** [comment]
thanks

**Pull request merged**